src/components/sections/general/form-sections/sync-checkbox-field.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { CheckboxField } from '@dhis2/ui'
+import { FormSection } from './form-section'
+
+export const SyncCheckboxField = ({ handleForm, syncElement }) => {
+    const { label, helpText, syncType } = syncElement
+    return (
+        <FormSection>
+            <CheckboxField
+                label={i18n.t('{{label}}', { label })}
+                helpText={i18n.t('{{helpText}}', { helpText })}
```

Comment:
Calling `i18n.t` here with only a variable doesn't do anything, the text needs to be translated before it is passed to this component.